### PR TITLE
Fixes #1571 [Added README File]

### LINKED ISSUE #1571: Add README.md file for Fibonacci_Number
I would like to contribute to GSSoC'2020 by making & adding a README.md file for given program

### PR DESCRIPTION
**Pull request by HardevKhandhar**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding README file for Fibonacci_Number
- 

#### Languages Used:
- English
- 

#### Files Added:
- README.md
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Fibonacci_Number/README.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+```
+F(n) = F(n - 1) + F(n - 2)
+```
+
+So, the Fibonacci Sequence becomes
+```
+0, 1, 1, 2, 3, 5, 8, 13, 21, 34, ...
+```
+and so on!
```
Comment (asha15) [inline Fibonacci_Number/README.md]: delete this line

File: Fibonacci_Number/README.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+Step 3: Initialize variables
+          a ← 1,
+          b ← 1,
+          i ← 2
+Step 4: Read n from user
+Step 5: Print a and b
+Step 6: Repeat until i < n
+           fib ← a + b
+           print fib
+           a ← b, b ← fib
```
Comment (asha15) [inline Fibonacci_Number/README.md]: take `b ← fib` into the next line

File: Fibonacci_Number/README.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+IF n is equal to 2
+   DISPLAY 1, 1
+
+IF n is greater than 2
+   a = 1,
+   b = 1,
+
+   DISPLAY a, b
+   REPEAT FOR 0 to n
```
Comment (asha15) [inline Fibonacci_Number/README.md]: recheck the logic

**Comment by asha15** [comment]
@HardevKhandhar  can you squash the commit then will go for merge

**Comment by HardevKhandhar** [comment]
Dear @jainaman224 Sir
This PR was merged under GSSoC 2020
Please add labels for difficulty level & GSSoC
My scores are not updated at the website. Kindly Help me with this
Thank you

**Comment by jainaman224** [comment]
Please update the PR template with issue number.

**Comment by HardevKhandhar** [comment]
PR template updated with issue number.
Done.

**Pull request merged**